### PR TITLE
Replace renderAsRichText() by pi_RTEcssText() method

### PR DESCRIPTION
**Pull request by djangones**
A fix for link handler links (t3://) not parsed anymore in Typo3 11, as discussed in #2372.

### CODE REVIEW

**Review by oliverklee — CHANGES_REQUESTED** [review]
We also need to adapt the call in line 2302 in `getOrganizersMarkerContent` and then can remove the `renderAsRichText` method.

**Comment by djangones** [comment]
Was not sure about removing that method completely but her you go...

File: Classes/FrontEnd/DefaultController.php [hunk 1]
```diff
@@ -2586,13 +2586,4 @@ protected function getConfigurationWithFlexForms(): Configuration
...
         return $this->configuration;
     }
-
-    private function renderAsRichText(string $rawData): string
-    {
-        $arguments = ['parseFuncTSPath' => 'lib.parseFunc_RTE'];
-        $childrenClosure = function () use ($rawData): string {
-            return \trim($rawData);
-        };
-        return HtmlViewHelper::renderStatic($arguments, $childrenClosure, new NullRenderingContext());
```
Comment (oliverklee) [inline Classes/FrontEnd/DefaultController.php:2596]: We probably then also can remove the `use` of `HtmlViewHelper` and `NullRenderingContext` now.

**Review by oliverklee — CHANGES_REQUESTED** [review]

**Review by oliverklee — APPROVED** [review]

**Review by oliverklee — APPROVED** [review]
Looks good (and works fine). Thanks for contributing, @djangones! :heart:

**Pull request merged**